session-3-jshell/README.md

```diff
@@ -4,8 +4,33 @@ Java REPL (Read-Eval-Print-Loop): previously called Kulla.
 
 The JShell API can be used via applications as well.
 
-Look for the the JShell REPL in the `JAVA_HOME/bin` folder:
+Windows Users
 
+    Look for the the JShell REPL in the `JAVA_HOME\bin` folder:
+
+    c:\... dir /W %JAVA_HOME%\bin\js*
```

Comment:
Could you please put blocks like this in back ticks (```) so it looks like 

```
c:\... dir /W %JAVA_HOME%\bin\js*
```